pkg/operator/operator.go

```diff
@@ -416,6 +434,53 @@ func (o *Operator) applyStatusInformers(a *apptypes.App, sequence int64, kotsKin
 	return nil
 }
 
+func (o *Operator) ResumeHelmStatusInformers() {
+	helmApps := helm.GetCache()
```

Comment:
Can't expose cache like this.  It must be behind a muted.

This can work instead
```
		for _, releaseName := range helm.GetCachedHelmApps() {
			release := helm.GetHelmApp(releaseName)
...
```